default.nix

```diff
@@ -49,4 +52,19 @@ pkgs.stdenv.mkDerivation {
     "fortify"
     "fortify3"
   ];
+
+  # Running Hyrise executables requires the LD_LIBRARY_PATH variable set. This does not
+  # happen by other hooks, therefore it happens manually here. Each of the derivations
+  # listed here provides a dynamically linked library.
+  # View the results by running `nix-shell --pure --run "echo $LD_LIBRARY_PATH"`.
+  shellHook = ''
+	export LD_LIBRARY_PATH="${pkgs.lib.makeLibraryPath [
```

Comment:
Can we also extend `$PATH` here to we don't have this long prefix in the nix call?

Reply:
Sadly, we already have to call nix-shell with the long prefix in order to execute this shellhook. So this won't work I guess

Reply:
Why would setting a PATH not work?

Reply:
That would definitly work, but as I understood your question is to avoid `/path/to/nix/nix-shell` and directly call `nix-shell`.

Reply:
Yes, and aren’t both connected?
Setting the PATH should allow us call `nix-shell` directly, right?

Curious why `clang` is found but `nix-shell` not.

Reply:
Ah, I see the issue. The point is that running `nix-shell` invokes the `shellHook` and sets the `PATH` with clang and the others. `nix-shell` however, needs to be in place before nix can do something. Adding `export PATH=$PATH:/path/to/nix/` would be called by `nix-shell`, so this does not have an effect.

Reply:
Got it!